packages/zowe-explorer/README.md

```diff
@@ -24,18 +23,25 @@ More information:
 
 ## Contents
 
-- [What's new in Zowe Explorer 1.10.0](#what's-new-in-zowe-explorer-1.10.0)
+- [What's new in Zowe Explorer 1.11.0](#whats-new-in-zowe-explorer-1.11.0)
 - [Prerequisites](#prerequisites)
 - [Getting Started](#getting-started)
+- [Sample Use Cases](#sample-use-cases)
 - [Credentials Security](#credentials-security)
 - [Usage Tips](#usage-tips)
 - [Extending Zowe Explorer](#extending-zowe-explorer)
 
-## What's new in Zowe Explorer 1.10.0
+## What's new in Zowe Explorer 1.11.0
 
 New features:
 
-- Added base profile support. You can now log in to the API Mediation Layer via Zowe CLI and use that token with a service profile in Zowe Explorer.
+- Added login and logout functions for base profiles. You can now log in to API Mediation Layer and generate a token for your base profile.
+
```

Comment:
This is where the note should go about the self-signed certificates issue that is documented in detail later on (the part that Richelle added).

Reply:
this is no longer needed with @jellypuno 's fix https://github.com/zowe/vscode-extension-for-zowe/pull/1101